perf/temp_findings.json

```diff
@@ -1,4003 +0,0 @@
-[
```

Comment:
why was this deleted? @IagoAbal ? @emjin ?

Reply:
I deleted it because given that all paths referred to Emma's HOME path then I assume this was committed by mistake. I warned and asked Emma to review the PR.